client/dashboard/sites/hosting-feature-gated-with-callout/upsell.tsx

```diff
@@ -2,41 +2,31 @@ import { __experimentalText as Text, Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { settings } from '@wordpress/icons';
 import { addQueryArgs } from '@wordpress/url';
-import { useEffect } from 'react';
-import { useAnalytics } from '../../app/analytics';
 import { Callout } from '../../components/callout';
 import illustrationUrl from './upsell-illustration.svg';
 import type { CalloutProps } from '../../components/callout/types';
 import type { Site } from '../../data/types';
 
-interface HostingFeatureUpsellProps {
-	site: Site;
-	tracksFeatureId: string;
-	icon?: CalloutProps[ 'icon' ];
-	image?: CalloutProps[ 'image' ];
-	title?: CalloutProps[ 'title' ];
-	description?: CalloutProps[ 'description' ];
+export interface UpsellCalloutProps {
```

Comment:
Nit: Since the context is already UpsellCallout, can we remove the upsell prefix? e.g.: upsellIcon -> icon.

Reply:
It's so that we can reuse the props here 😄 

https://github.com/Automattic/wp-calypso/pull/104772/files#diff-3836c80a6897e920f283f0356e062278d1f34c4a358530aa0e885a3fccf7554cR10